polkadot-parachains/integritee-runtime/src/lib.rs

```diff
@@ -550,6 +552,140 @@ impl pallet_treasury::Config for Runtime {
 	type WeightInfo = weights::pallet_treasury::WeightInfo<Runtime>;
 }
 
+// Council
+pub type CouncilInstance = pallet_collective::Instance1;
+
+parameter_types! {
+	pub const CouncilMotionDuration: BlockNumber = prod_or_fast!(3 * DAYS, 2 * MINUTES);
+	pub const CouncilMaxProposals: u32 = 100;
+	pub const CouncilMaxMembers: u32 = 100;
+}
+
+impl pallet_collective::Config<CouncilInstance> for Runtime {
+	type Origin = Origin;
+	type Proposal = Call;
+	type Event = Event;
+	type MotionDuration = CouncilMotionDuration;
+	type MaxProposals = CouncilMaxProposals;
+	type MaxMembers = CouncilMaxMembers;
+	type DefaultVote = pallet_collective::PrimeDefaultVote;
+	type WeightInfo = pallet_collective::weights::SubstrateWeight<Runtime>;
+}
+
+pub type EnsureRootOrAllCouncil = EnsureOneOf<
+	EnsureRoot<AccountId>,
+	pallet_collective::EnsureProportionAtLeast<AccountId, CouncilInstance, 1, 1>,
+>;
+
+pub type EnsureRootOrHalfCouncil = EnsureOneOf<
+	EnsureRoot<AccountId>,
+	pallet_collective::EnsureProportionAtLeast<AccountId, CouncilInstance, 1, 2>,
+>;
+
+pub type EnsureRootOrTwoThirdsCouncil = EnsureOneOf<
+	EnsureRoot<AccountId>,
+	pallet_collective::EnsureProportionAtLeast<AccountId, CouncilInstance, 2, 3>,
+>;
+
+pub type EnsureRootOrThreeFifthCouncil = EnsureOneOf<
+	EnsureRoot<AccountId>,
+	pallet_collective::EnsureProportionAtLeast<AccountId, CouncilInstance, 3, 5>,
+>;
+
+pub type EnsureRootOrOneCouncil =
+	EnsureOneOf<EnsureRoot<AccountId>, pallet_collective::EnsureMember<AccountId, CouncilInstance>>;
+
+// Technical Committee
+pub type TechnicalCommitteeInstance = pallet_collective::Instance2;
+
+parameter_types! {
+	pub const TechnicalMotionDuration: BlockNumber = prod_or_fast!(3 * DAYS, 2 * MINUTES);
+	pub const TechnicalMaxProposals: u32 = 100;
+	pub const TechnicalMaxMembers: u32 = 100;
+}
+
+impl pallet_collective::Config<TechnicalCommitteeInstance> for Runtime {
+	type Origin = Origin;
+	type Event = Event;
+	type Proposal = Call;
+	/// The maximum amount of time (in blocks) for technical committee members to vote on motions.
+	/// Motions may end in fewer blocks if enough votes are cast to determine the result.
+	type MotionDuration = TechnicalMotionDuration;
+	/// The maximum number of Proposlas that can be open in the technical committee at once.
+	type MaxProposals = TechnicalMaxProposals;
+	/// The maximum number of technical committee members.
+	type MaxMembers = TechnicalMaxMembers;
+	type DefaultVote = pallet_collective::MoreThanMajorityThenPrimeDefaultVote;
+	type WeightInfo = pallet_collective::weights::SubstrateWeight<Runtime>;
```

Comment:
Is this a todo? The substrate-weights are too small for parachains.

Reply:
Ah. I was hoping to get away with it because moonriver also uses the substrate weights (https://github.com/PureStake/moonbeam/blob/master/runtime/moonriver/src/lib.rs#L459).

I'll ask Zoltan to do the benchmarkign.

Reply:
Cool thanks! I think using substrate weights is acceptable in a standalone node, but even then not good practice.

Reply:
Weights are finally added. Is it fine like this?